### PR TITLE
"not" missing

### PR DESCRIPTION
**Pull request by wombatonfire**
"If you suspect that any of your federated partners may be using Direct Federation or have federated with any online or hybrid organization, we suggest you send them a communication about this as you prepare to complete your migration to the cloud."

should be

"If you suspect that any of your federated partners may be using Direct Federation or have **not** federated with any online or hybrid organization, we suggest you send them a communication about this as you prepare to complete your migration to the cloud."

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbHybrid/hybrid/cloud-consolidation-disabling-hybrid.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ These steps should be done together as a unit. Details are provided below. In ad
...
+Any federated organizations that are using the older Direct Federation model (also known as Allowed Partner Server) will need to update their allowed domain entries for their organization to remove the proxy FQDN. This legacy federation model is not based on DNS SRV records, so such a configuration will become out of date once your organization moves to the cloud. </li><li>Any federated organization that does not have an enabled hosting provider for sipfed.online.lync.<span>com will need to update their configuration to enable that. This situation is only possible if the federated organization is purely on premises and has never federated with any hybrid or online tenant. In such a case, federation with these organizations will not work until they enable their hosting provider.</li></ul>If you suspect that any of your federated partners may be using Direct Federation or have not federated with any online or hybrid organization, we suggest you send them a communication about this as you prepare to complete your migration to the cloud.
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/cloud-consolidation-disabling-hybrid.md:40]:
```suggestion
Any federated organizations that are using the older Direct Federation model (also known as Allowed Partner Server) will need to update their allowed domain entries for their organization to remove the proxy FQDN. This legacy federation model is not based on DNS SRV records, so such a configuration will become out of date once your organization moves to the cloud. </li><li>Any federated organization that does not have an enabled hosting provider for sipfed.online.lync.<span>com will need to update their configuration to enable that. This situation is only possible if the federated organization is purely on-premises and has never federated with any hybrid or online tenant. In such a case, federation with these organizations will not work until they enable their hosting provider.</li></ul>If you suspect that any of your federated partners may be using Direct Federation or have not federated with any online or hybrid organization, we suggest you send them a communication about this as you prepare to complete your migration to the cloud.
```

Comment (JohanFreelancer9) [reply Skype/SfbHybrid/hybrid/cloud-consolidation-disabling-hybrid.md:40]: Please also change "on premises" to "on-premises".

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**